### PR TITLE
Update CI to check old db upgrade

### PR DESCRIPTION
**Pull request by as6325400**
## issue
https://github.com/DOMjudge/domjudge/issues/2577

Add version 7.3.3 data to check `dj_setup_database` upgrade.

### CODE REVIEW

File: .github/workflows/database-upgrade.yml
Comment (vmcj) [inline .github/workflows/database-upgrade.yml:33]: I think we already install those tools in the `domjudge/gitlabci:24.04` and otherwise please PR that one first.

File: .github/workflows/database-upgrade.yml
Comment (vmcj) [inline .github/workflows/database-upgrade.yml:34]: I think `GHA` will always first boot the service containers before booting the container, so is this step actually needed?

File: .github/workflows/database-upgrade.yml
Comment (vmcj) [inline .github/workflows/database-upgrade.yml:44]: Missing newline.

File: .github/workflows/database-upgrade.yml
Comment (vmcj) [inline .github/workflows/database-upgrade.yml:42]: Can you also run the download part from `webstandard.sh` and make sure that none of the pages result in a `HTTP500`? That's what most of the time happens.

Comment (as6325400) [reply .github/workflows/database-upgrade.yml:34]: What test and role should I use to execute webstandard.sh? Or should I modify the script to add a new test parameter that only checks whether any page returns a 500 error?

Comment (vmcj) [reply .github/workflows/database-upgrade.yml:34]: I would download as admin with a team role and I would just not run the `WCAG` or `W3C` tests but only do the downloading part with `wget`.

Comment (as6325400) [reply .github/workflows/database-upgrade.yml:34]: If I only run webstandard.sh after the database upgrade is complete and use the admin role, will that be sufficient?

Comment (vmcj) [reply .github/workflows/database-upgrade.yml:34]: Also fine by me.

Comment (as6325400) [reply .github/workflows/database-upgrade.yml:34]: Okay, it's been added. Is this okay?

File: .github/workflows/database-upgrade.yml
Comment (vmcj) [inline .github/workflows/database-upgrade.yml:40]: Your editor doesn't add the EOF char.

File: .github/jobs/webstandard.sh
Comment (vmcj) [inline .github/jobs/webstandard.sh:95]: Reviewing would be much easier if you exit here early.

So if TEST==none: exit(number_of_errors)

**Review by vmcj — APPROVED** [review]
I'll wait for approval of someone else also but LGTM.

One last thing, can you describe in the commit message what things you did in the dump? So I assume you logged in as admin, created a user etc. would be good to clearly describe what was done so if needed we can extend this in the future.

Would be good to have as minimal:
- Created a team,
- Submitted as a team
- Submitted something wrong as a team
- Judged both submissions
- Resubmit something as an admin
- Send clarification as jury
- Send clarification as team
- Change a configuration option from default to something else
- Change an executable
- Do a rejudging of a single submission

If something wasn't done feel free to add it to the original issue so we can work on it in the future.

File: .github/workflows/database-upgrade.yml
Comment (nickygerritsen) [inline .github/workflows/database-upgrade.yml:30]:
```suggestion
      - name: Import Database
```

File: .github/workflows/database-upgrade.yml
Comment (nickygerritsen) [inline .github/workflows/database-upgrade.yml:31]:
```suggestion
        run: mysql -hsqlserver -uroot -proot < .github/jobs/data/dj733.sql
```
I don't think we need the quotes

File: .github/workflows/database-upgrade.yml
Comment (nickygerritsen) [inline .github/workflows/database-upgrade.yml:32]:
```suggestion
      - name: Upgrade DOMjudge
```

File: .github/workflows/database-upgrade.yml
Comment (nickygerritsen) [inline .github/workflows/database-upgrade.yml:34]:
```suggestion
      - name: Setting initial Admin Password
```

File: .github/workflows/database-upgrade.yml
Comment (nickygerritsen) [inline .github/workflows/database-upgrade.yml:37]:
```suggestion
        run: mysql -hsqlserver -uroot -proot -e "SHOW TABLES FROM domjudge;"
```

**Review by nickygerritsen — APPROVED** [review]
Some small comments but besides that it looks good.

**Comment by as6325400** [comment]
Changes made and explained the actions I performed in the dump.

**Pull request merged**
